cartography/util.py

```diff
@@ -58,3 +63,25 @@ def timed(*args, **kwargs):
             return method(*args, **kwargs)
 
     return timed
+
+
+def aws_handle_regions(func):
+    ERROR_CODES = [
+        'AccessDeniedException',
+        'UnrecognizedClientException',
+        'InvalidClientTokenId',
+        'AuthFailure',
+    ]
+
+    def inner_function(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except botocore.exceptions.ClientError as e:
+            # The account is not authorized to use this service in this region
+            # so we can continue without raising an exception
+            if e.response['Error']['Code'] in ERROR_CODES:
+                logger.warn("{} in this region. Skipping...".format(e.response['Error']['Message']))
```

Comment:
Let's use `logger.warning()` now; I'm getting a message in Pycharm that `logger.warn()` is going to be deprecated